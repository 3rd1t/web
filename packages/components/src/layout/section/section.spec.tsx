import React from "react"
import { render } from "@testing-library/react"
import Section from "./section"

describe("Section", () => {
  describe("without domain", () => {
    it("renders correctly", () => {
      const { baseElement } = render(
        <Section bg="bg-white" id="hello">
          <span>Hello World</span>
        </Section>,
      )
      expect(baseElement).toBeTruthy()
    })
  })
})
