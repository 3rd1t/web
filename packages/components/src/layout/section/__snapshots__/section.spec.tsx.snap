// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section without domain renders correctly 1`] = `
<section
  className="bg-white"
  id="hello"
>
  <div>
    <div
      className="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
    >
      <span>
        Hello World
      </span>
    </div>
  </div>
</section>
`;
